projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts

```diff
@@ -84,6 +90,10 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
     this.keywordsService.toggleKeyword(keyword);
   }
 
+  onAllSeriesCheckboxClick(event: any) {
```

Comment:
we dont seem to care about the event, therefore it can be removed as a parameter